CityOfDetroit/bloom
# Partial upstream merge

## Issue

- Addresses #293 

## Description

Merges upstream changes up to commit `4b6718eb94e458fa4d980c5062fdfa1485b55603`.

Include node upgrade to v 14.

I also fixed a backend issue where the type of `UnitsSummary.unitType` was changed to `Id`, when it should be type `UnitTypeDto`. This should be changed upstream soon.

## Type of change


## How Can This Be Tested/Reviewed?

You'll have to upgrade node to v 14. After that I clicked around both portals to check for errors and verify functionality.


## Checklist:
